packages/paste-core/components/minimizable-dialog/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@twilio-paste/minimizable-dialog",
+  "version": "0.0.0",
+  "category": "overlays",
+  "status": "alpha",
```

Comment:
chore: technically not alpha. Stage 1 yes, but this is prod spec

Reply:
oof then I should update that on callout too